schema/vrs.json

```diff
@@ -119,11 +123,12 @@
             "type": {
                "type": "string",
                "const": "Haplotype",
+               "default": "Haplotype",
                "description": "MUST be \"Haplotype\""
             },
             "members": {
                "type": "array",
-               "minItems": 1,
+               "minItems": 2,
```

Comment:
Probably has been discussed - but what about haploid genomes/chromosomes?

Reply:
This `Haplotype` data type is a molecular variation of in-cis members that are comprised of Alleles. So it should work for any system haploid, diploid, etc... We made this specific change to avoid the confusion of treating a `Haplotype` of 1 member `Allele` to be the same thing as a single `Allele`.  Since both `Allele` and `Haplotype` are subtypes of `MolecularVariation` you should always be able to use `Allele` where you would otherwise use a `Haplotype` of 1 member, thus we made haplotype 2...m to avoid the potential for dual representation of a single allele.

Reply:
@mbaudis I just realized that I may have misunderstood your question.  To be clear, the `minItems` constraint is about the the cardinality of _in-cis_ allele members for the Haplotype being defined. It is not a ploidy count or anything like that. Haplotypes by definition are haploid.  We are using the Genotype class to deal with genome ploidy representation. Refer to the `Genotype.copies` for this attribute (btw - we are considering changing the attribute name from copies to something that more clearly represents the number of _in-trans_ occurrences of each`Gentotype.member`.  Let us know if this resolves your issue. If not, please restate so we can be sure to understand any additional concerns.